cypress/support/index.d.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
?

Reply:
There's something weird where `eslint` says `Chainable` is not used (even though it is). Even in the lib `cypress-testing-library` they added it for the same reason.